### PR TITLE
Issue/9

### PR DESCRIPTION
**Pull request by tsbiberdorf**
Please review the RS485 loop back fix as well as adding the Supervisor Task to the project

### CODE REVIEW

**Comment by sWahlinRhoades** [comment]
issue/8 was merged into this branch so merge issue/8 before this one into develop

File: source/Peripheral/RS485/RS485Sensor.c [hunk 1]
```diff
@@ -100,105 +104,164 @@ static SemaphoreHandle_t tl_Sensor485SendMsgSem = NULL;
...
+		// set transmit bit on RS485 port
+		GPIO_PortSet(BOARD_SENSOR_RS485_DIR_GPIO,1<<BOARD_SENSOR_RS485_DIR_PIN);
+
+		// now enable the transmit IRQ handler to send data
+		UART_EnableInterrupts(RS485_UART_PERIPHERAL, kUART_TxDataRegEmptyInterruptEnable | kUART_TransmissionCompleteInterruptEnable | kUART_RxDataRegFullInterruptEnable | kUART_RxOverrunInterruptEnable);
+	}
+	return;
+}
+
+static void LoopbackTransmitDataMode()
```
Comment (sWahlinRhoades) [inline source/Peripheral/RS485/RS485Sensor.c:156]: missing a function header

File: source/Supervisor/SupervisorTask.c [hunk 2]
```diff
@@ -0,0 +1,109 @@
+/**
+ * @ingroup groupSupervisor
+ *
+ * @file SupervisorTask.c
+ *
+ * Receive data from a freeRTOS Queue from the Sensor RS485 task.
```
Comment (sWahlinRhoades) [inline source/Supervisor/SupervisorTask.c:6]: the description needs to be tagged with @details it isn't showing up in doxygen

**Comment by sWahlinRhoades** [comment]
Other than the minor issues with doxygen I think this branch is fine

File: source/Peripheral/RS485/RS485Sensor.c (hunk 1 above)
Comment (tsbiberdorf) [reply source/Peripheral/RS485/RS485Sensor.c:160]: updated the tags

File: source/Supervisor/SupervisorTask.c (hunk 2 above)
Comment (tsbiberdorf) [reply source/Supervisor/SupervisorTask.c:6]: updated the tags

**Comment by sWahlinRhoades** [comment]
looks good to me

**Pull request merged**